bindings/generator/templates/binding_electron_meths.rs.j2

```diff
@@ -479,7 +488,33 @@ fn {{ variant_rs_to_js_function_name(variant) }}<'a>(
 }
 {% endmacro -%}
 
+{%- macro enum_js_to_rs_function_name(type) %}enum_{{ type.name | pascal2snake }}_js_to_rs{% endmacro -%}
```

Comment:
We currently use `type.name | lower` for `struct_js_to_rs_function_name/struct_rs_to_js_function_name/variant_js_to_rs_function_name/variant_rs_to_js_function_name`

If you really think it is worth it to add `pascal2snake` instead of just lowercasing, then we should also correct those functions

Reply:
done